src/components/FilterModal/FilterModal.scss

```diff
@@ -0,0 +1,84 @@
+@import '../../assets/styles/colors';
+
+.filters {
```

Comment:
Good job with SASS and BEM. I'd be grateful if you'd share with others, who may not know about it yet, how it's working!